.github/workflows/test.yml

```diff
@@ -0,0 +1,44 @@
+name: Test Python open payments sdk f
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Could you also run an audit as part of the flow?

<img width="815" height="416" alt="image" src="https://github.com/user-attachments/assets/e367aaad-69a6-4bdc-8feb-9a466efb57d7" />